### ISSUE TITLE
Fix XML tags in chapter 9

### ISSUE THREAD

**Issue by mateunho** [debian-handbook-pl/pl-PL#67]
Refers to build break caused by improper tags.

**Comment by annldzpl** [comment]
Could you check if it builds correctly now?

**Comment by mateunho** [comment]
Nope.

```
    Merging pl-PL/09_unix-services.po >> en-US/09_unix-services.xml -> tmp/pl-PL/xml_tmp//09_unix-services.xml

mismatched tag at line 4, column 325, byte 492:
<!DOCTYPE para PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
]>
<para processed="1">Init uruchamia kilka procesów, następujące polecenia z pliku <filename>/etc/inittab</filename>. Pierwszy uruchamiany program(który nawiązuje do kroku <emphasis>sysinit</emphasis> ) jest to  <command>/etc/init.d/rcS</command>, skrypt, który uruchamia wszystkie programy w <filename>/etc/rcS.d/</filename></primary></indexterm> <indexterm><primary><filename>rcS.d</filename></primary></indexterm></para>
====================================================================================================================================================================================================================================================================================================================================^
 at /usr/lib/perl5/XML/Parser.pm line 187.
ERROR: failed to build HTML output for pl-PL
```

> Look up the quoted [note](https://github.com/debian-handbook-pl/pl-PL/issues/66#issuecomment-75398336)

**Comment by mateunho** [comment]
Oh, by the way, next time commit changes on branch responding to the title of mentioned chapter, please. :pray:

**Comment by mateunho** [comment]
Fixed with e04750c1d526b71b77ace08ae851cb3a5326b922
